package.json

```diff
@@ -140,6 +141,7 @@
     "sass": "^1.51.0",
     "sass-loader": "^10.2",
     "typescript": "^4.6.4",
+    "vite-plugin-vue2": "^2.0.0",
```

Comment:
already?

Reply:
if someone wants to write unit test for the vue component, he will need `@vue/test-utils` and `vite-plugin-vue2`

Reply:
could you remove the 2 packages you added? Your tests pass without them
<img width="306" alt="Capture d’écran 2022-05-06 à 4 58 25 PM" src="https://user-images.githubusercontent.com/9987732/167159148-3e00f5e1-03e2-4c45-a685-db21c182039e.png">